tests/test_stabilize_weights_dmc.py

```diff
@@ -0,0 +1,92 @@
+# This must be done BEFORE importing numpy or anything else.
+# Therefore it must be in your main script.
+import os
+os.environ["MKL_NUM_THREADS"] = "1"
+os.environ["NUMEXPR_NUM_THREADS"] = "1"
+os.environ["OMP_NUM_THREADS"] = "1"
+import sys
+import numpy as np
+import pyqmc.testwf as testwf
+import pytest
+import pyblock.pd_utils as pyblock
+
+def use_parsl():
```

Comment:
This does not work with the continuous integration testing, and this seems a bit too long to have as a standard test. I think this should probably be implemented as an example.